spec/lib/axe/core_spec.rb

```diff
@@ -6,10 +6,13 @@ module Axe
     subject(:core) { described_class.new(page) }
     let(:page) { spy('page', evaluate_script: false) }
 
-    describe "initialize" do
+		describe "initialize" do
```

Comment:
more weird indentation

Reply:
Same comment as before.

Reply:
Yep, same. There are tabs in them thar files!

![image](https://user-images.githubusercontent.com/119972/44743884-b23c8e80-aad1-11e8-94b6-89591df7874b.png)